packages/widgets/src/Transfer/Transfer.stories.js

```diff
@@ -118,36 +94,47 @@ const options = [
     },
 ]
 
-export default { title: 'Transfer', decorators: [statefulDecorator()] }
+/* eslint-disable */
```

Comment:
It'd be nice to not have to disable ESLint here. If there are valid reasons to disable the linter I would expect just a specific rule to be disabled, and probably a comment to explain why it wasn't possible to produce this code in a manner that is compliant with our ESLint rules.